Marlin/Configuration_adv.h

```diff
@@ -357,10 +357,10 @@
 // Steppers will shut down DEFAULT_STEPPER_DEACTIVE_TIME seconds after the last move when DISABLE_INACTIVE_? is true.
 // Time can be set by M18 and M84.
 #define DEFAULT_STEPPER_DEACTIVE_TIME 120
-#define DISABLE_INACTIVE_X true
-#define DISABLE_INACTIVE_Y true
-#define DISABLE_INACTIVE_Z true  // set to false if the nozzle will fall down on your printed part when print has finished.
-#define DISABLE_INACTIVE_E true
+#define DISABLE_INACTIVE_X false
+#define DISABLE_INACTIVE_Y false
+#define DISABLE_INACTIVE_Z false  // set to false if the nozzle will fall down on your printed part when print has finished.
+#define DISABLE_INACTIVE_E false
 
```

Comment:
Ahh - i see.
`DISABLE_INACTIVE_Z` in RCBugFix is not comparable to `DISABLE_E`. In RCBug fix we have both `DISABLE_E` and `DISABLE_INACTIVE_Z`. The first disables the stepper immediately when there is no move is in the planner for that axis. The later disables the stepper `stepper_inactive_time` ms after the last move if `stepper_inactive_time` is not zero.
`DISABLE_Z` is still false by default.
If you want to disable shutting down the steppers after `stepper_inactive_time` is over set `DEFAULT_STEPPER_DEACTIVE_TIME` to 0 in __your__ configuration.

You are right direct calls of M84 are unaffected.

Yes there was a bug in that area of 1.0.2. Steppers where disabled ether immediately or never.